scripts/spellcheck.ts

```diff
@@ -0,0 +1,65 @@
+import { devOps, cli, childProcess } from '@azure/avocado';
+import { ExecOptions } from 'child_process';
+
+const logError = (msg: string) => {
+  const lines = msg.split('\n');
+  for (const line of lines) {
+    console.log(`##vso[task.logissue type=error]${line}`);
+  }
+}
+
+const verboseExec = async (commandLine: string, options: ExecOptions = {}) => {
+  console.log(commandLine);
+  let result: any = {};
+  try {
+    result = await childProcess.exec(commandLine, options);
+  } catch (e) {
+    result = e;
+ }
+
+ if (!result.code) {
+   return 0;
+ }
+
+ if (result.stdout) {
+   logError(result.stdout);
+ }
+ if (result.stderr) {
+   console.error(result.stderr);
+ }
+ return result.code;
+}
+
+const main = async () => {
+  const config = cli.defaultConfig();
+  const pr = await devOps.createPullRequestProperties(config);
```

Comment:
why need dependency on avocado? how about get changed files by github directly?

Reply:
That's kind of duplicated of logics. Get that changed file list require about 100 lines of code and I don't want to copy/paste that piece of code everywhere.